Error with json

I've got this error : 

```
$ ./jam
Email: xxxxxxxxxx@gmail.com
Password: ***************
2017/01/26 23:13:56 json: cannot unmarshal number 17748 into Go value of type uint8
```

Any idea ?

```
./jam -debug
Email: xxxxxxxx@gmail.com
Password: ***************
panic: json: cannot unmarshal number 17748 into Go value of type uint8

goroutine 1 [running]:
panic(0x6b93e0, 0xc4208b0900)
        /home/nlamirault/Apps/go/src/runtime/panic.go:500 +0x1a1
main.checkErr(0xa59e00, 0xc4208b0900)
        /home/nlamirault/Apps/golang/src/github.com/budkin/jam/main.go:78 +0xc4
main.refreshLibrary()
        /home/nlamirault/Apps/golang/src/github.com/budkin/jam/lib.go:49 +0x67
main.checkCreds()
        /home/nlamirault/Apps/golang/src/github.com/budkin/jam/auth.go:58 +0x6e
main.main()
        /home/nlamirault/Apps/golang/src/github.com/budkin/jam/main.go:68 +0xb3
```

--- a/lib.go
+++ b/lib.go
@@ -31,7 +31,7 @@ import (
 type bTrack struct {
 	//AlbumArtist    string
 	DiscNumber     uint8
-	TrackNumber    uint8
+	TrackNumber    uint32
 	DurationMillis string
 	EstimatedSize  string
 	ID             string

